rocAL/rocAL_pybind/examples/tf_petsTrainingExample/download_and_preprocess_dataset.sh

```diff
@@ -14,8 +13,10 @@ tar xzvf images.tar.gz
 tar xzvf annotations.tar.gz
 mkdir tf_pets_records
 git clone https://github.com/swetha097/rocALmodels.git
+cd rocALmodels
```

Comment:
Please check if these changes are needed in this PR

Reply:
A change from ToT not reflected in Shobana's fork